contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -545,6 +545,29 @@ namespace eosiosystem {
       asset stake_change;
    };
 
+   struct action_return_sellram {
+      name account;
+      asset quantity;
+      int64_t bytes;
+      int64_t ram_bytes;
```

Comment:
Not a big deal. Those two fields `bytes` and `ram_bytes` are confusing at the first glance. `ram_bytes` cannot be changed; can `bytes` be renamed to `bytes_sold`? Or just add some comments here.

Reply:
👍 `bytes_sold`  is a good idea, can easily rename the struct to that

@ericpassmore would you be able to push that update

Reply:
opened issue and PR with code changes.